pkg/identityserver/store/pagination.go

```diff
@@ -31,6 +31,14 @@ type PaginationOptions struct {
 	total  *uint64
 }
 
+// WithLimit checks if the limit is set, and returns the set limit or the default.
+func WithLimit(setLimit uint32, defaultLimit uint32) uint32 {
```

Comment:
` func WithLimit(set, default uint32) uint32` is more concise.

Reply:
`default` is a reserved keyword, but I used `def` then